catalyst_voices/packages/internal/catalyst_voices_repositories/test/src/document/document_repository_test.dart

```diff
@@ -23,6 +23,7 @@ void main() {
   late DraftDataSource draftsSource;
   late SignedDocumentDataSource localDocuments;
   late DocumentDataRemoteSource remoteDocuments;
+  late DocumentFavoriteSource favorite;
```

Comment:
I'd keep it in sync with others here:
```suggestion
  late DocumentFavoriteSource favoriteDocuments;
```